### PR TITLE
Project Initialization

### PR DESCRIPTION
**Pull request by Lubandeo**
## Description
Initialized the project and the repository

## Functionality
1) Initialize and share the repository with teammates and mentors
2) Create the project
3) Add correct gitIgnore
4) Clean Architecture
5) Add lint

## Package Structure
image

### CODE REVIEW

File: data/build.gradle [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+}
+
```
Comment (SartiMau) [inline data/build.gradle:31]: did you miss ktlint?

File: di/build.gradle [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+}
+
```
Comment (SartiMau) [inline di/build.gradle:31]: did you miss ktlint?

File: domain/build.gradle [hunk 3]
```diff
@@ -0,0 +1,9 @@
+plugins {
+    id 'java-library'
+    id 'kotlin'
+}
+
```
Comment (SartiMau) [inline domain/build.gradle:5]: did you miss ktlint?

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: domain/build.gradle (hunk 3 above)
Comment (julian-mainardis-globant) [reply domain/build.gradle:5]: done

File: di/build.gradle (hunk 2 above)
Comment (julian-mainardis-globant) [reply di/build.gradle:31]: done

File: data/build.gradle (hunk 1 above)
Comment (julian-mainardis-globant) [reply data/build.gradle:31]: done

**Pull request closed without merging**